server/node.js

```diff
@@ -0,0 +1,39 @@
+require('newrelic');
+const http = require('http');
+const path = require('path');
+const Promise = require('bluebird');
+const fs = Promise.promisifyAll(require('fs'));
+const db = require('../db/database.js');
+
+http.createServer((req, res) => {
+  if (req.url === '/') {
+    fs.readFileAsync(path.join(__dirname, '../client/dist/index.html'), 'utf8')
+      .then((data) => {
+        res.writeHead(200, { 'Content-Type': 'text/html' });
+        res.end(data);
+      })
+      .catch(() => {
+        res.writeHead(400);
+        res.end();
+      });
+  } else if (req.url.match('.css')) {
+    const cssStream = fs.createReadStream(path.join(__dirname, '../client/dist/styles.css'), 'utf8');
+    res.writeHead(200, { 'Content-Type': 'text/css' });
+    cssStream.pipe(res);
+  } else if (req.url.match('.js')) {
+    const jsStream = fs.createReadStream(path.join(__dirname, '../client/dist/bundle.js'), 'utf8');
+    res.writeHead(200, { 'Content-Type': 'text/javascript' });
+    jsStream.pipe(res);
+  } else if (req.url.match(/\/restaurants\/Restaurant%20\d{8}$/)) {
+    const id = req.url.slice(26);
+    db.find({ name: `Restaurant ${id}` }, (err, data) => {
+      if (err) {
```

Comment:
Might be better to change db.find() to something more specific so it's clear what you are 'finding' and that you are not simply calling Model.find() right here in this file - was a little confusing to follow at first.